rackcorp/resourceRackcorpServer.go

```diff
@@ -204,7 +210,7 @@ func resourceRackcorpServer() *schema.Resource {
 				Elem:     nicSchemaElement(),
 			},
 			"device_id": {
-				Type:     schema.TypeString,
+				Type:     schema.TypeInt,
```

Comment:
Again, string was chosen to keep the ID opaque. The consumer of this library doesn't need to perform integer arithmetic or greater than comparisons on the ID, it just needs to pass back the same value it received, or reference it elsewhere. If Rackcorp later change the format in their API it should not affect the consumer.  Our json encoding can treat it as string or int as needed to satisfy the on-the-wire requirements but this can remain hidden from the end user.